backend/index.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let sessionCounter = 0;
+let counter;
+
+try {
+  fs.existsSync("temp/my-counter-visited-times.txt")
+    ? (counter = fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
+    : (counter = 0);
```

Comment:
In general the better syntax would be like this (BTW you forgot to convert it to a number, always keep the value of the variable the same type, js is dynamic but it's always better to pretend that it's not for production code to avoid bugs :P )
```suggestion
  counter = fs.existsSync("temp/my-counter-visited-times.txt")
    ? Number(fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
    : 0;
```

But for this case I'd write
```suggestion
  if (fs.existsSync("temp/my-counter-visited-times.txt"))
    counter = fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"));
  }
```
and above, on the line 7 `let counter;` -> `let counter = 0;`